modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -6,55 +6,50 @@
 [id="nw-ingress-converting-http-header-case_{context}"]
 = Converting HTTP header case
 
-HAProxy lowercases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+HAProxy lowercases HTTP header names by default; for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
 
 [IMPORTANT]
 ====
-Because {product-title} includes HAProxy 2.8, be sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading.
+{product-title} includes HAProxy 2.8. If you want to upgrade to this version of the web-based load balancer, ensure that add `spec.httpHeaders.headerNameCaseAdjustments` to your cluster's configuration file.
 ====
 
+As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by setting the `HeaderNameCaseAdjustments` field in the Ingress Controller YAML file.
+
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
 
-As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by setting the `HeaderNameCaseAdjustments` field in the Ingress Controller YAML file.
+* Capitalize an HTTP header by using the `oc patch` command.
 
-* Specify an HTTP header to be capitalized by entering the `oc patch` command.
-
-. Enter the `oc patch` command to change the HTTP `host` header to `Host`:
+. Enter the `oc patch` command to change the HTTP header from `host` to `Host`:
```

Comment:
I think that if this is meant to be a substep, this should use a level 2 ordered list item, even if the parent list item is unordered. I don't know if there's a style guide ruling on this but it would just seem a bit confusing imo to use level one items in this context:
```suggestion
.. Enter the `oc patch` command to change the HTTP header from `host` to `Host`:
```

If for some reason the formatting doesn't work when you change it, then this can be left as is.

Reply:
:heavy_check_mark: